### PR TITLE
rootpy-foss-2016a-python-2.7.11

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

**Comment by ysagon** [comment]
This version shouldn't work before this PR get merged #4239

File: easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ('ROOT','v6.08.02','-Python-%(pyver)s'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s')
+]
+
+options = {'modulename': 'os'}
+
+#sanity_check_paths = {
+#    'files': ['bin/quast.py'],
+#    'dirs': [''],
+#}
```
Comment (boegel) [inline easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb]: @ysagon This should be fixed, to something like:

```python
sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```

of course, it other important files are also installed, they can be listed here too

File: easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = ['https://github.com/rootpy/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('ROOT','v6.08.02','-Python-%(pyver)s'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s')
+]
+
+options = {'modulename': 'os'}
```
Comment (boegel) [inline easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb]: this is a hack, it should not be needed as long as `import rootpy` works...

**Comment by ysagon** [comment]
Hello, I would like to go on with this PR, but I'm not sure to understand the error on Travis.

**Comment by boegel** [comment]
@ysagon The problem is that `matplotlib` v1.5.1 depends on `freetype` v2.6.2, while `ROOT` v6.08.02 depends on `freetype` v2.6.3.

The error message somehow mentions a bunch of other unrelated stuff too, which is why it's confusing, I'll see if we can get that fixed.

To fix this, you can change the `matplotlib` dependency to the one that also uses `freetype` v2.6.3:

```python
('matplotlib', '1.5.1', '-freetype-2.6.3'),
```

File: easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ('ROOT','v6.08.02','-Python-%(pyver)s'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s')
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+moduleclass = 'bio'
```
Comment (boegel) [inline easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb]: @ysagon This looks wrong, this is not bioinformatics software?

File: easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/rootpy/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('ROOT','v6.08.02','-Python-%(pyver)s'),
+    ('matplotlib', '1.5.1', '-freetype-2.6.3')
```
Comment (boegel) [inline easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb]: @ysagon You'll need to use `versionsuffix + '-freetype-2.6.3'` here for matplotlib

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3b3cfc69c366f143b754c0d3c740abd5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c796a48319ac97703e33f6d23634a3c5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ysagon!

**Pull request merged**